src/pages/[platform]/tools/libraries/configure-categories/index.mdx

```diff
@@ -26,10 +19,207 @@ export function getStaticProps(context) {
   };
 }
 
-import js0 from '/src/fragments/lib-v1/client-configuration/js/js-configuration.mdx'; 
+<InlineFilter filters={['javascript', 'react-native', 'angular', 'nextjs', 'vue','react']}>
 
-<Fragments fragments={{ javascript: js0, angular: js0, nextjs: js0, react: js0, vue: js0 }} />
+When using the Amplify CLI, the **aws-exports.js** file gets created and updated automatically for you based upon the resources you have added and configured.
```

Comment:
I might be wrong but I think the `aws-exports.js` got renamed to `amplifyConfiguration`.

Reply:
Good catch ty